.github/workflows/ci-tests.yml

```diff
@@ -0,0 +1,82 @@
+name: Build and test
+
+on:
+    # Only on pushes to main or one of the release branches we build on push
+    push:
+        branches:
+            - main
+            - "[0-9].[0-9]+-branch"
+        tags:
+    # Build pull requests
+    pull_request:
+
+jobs:
+    test:
+        strategy:
+            matrix:
+                py:
+                    - "3.7"
+                    - "3.8"
+                    - "3.9"
+                    - "3.10"
+                    - "3.11"
+                    # - "pypy-3.8"
```

Comment:
Comment to explain why we are skipping?

Reply:
Thanks for noticing, I just forgot to uncomment it while throwing things over the wall commented.